MetasysServices/MetasysClient.cs

```diff
@@ -512,10 +513,12 @@ public async Task<IEnumerable<MetasysObject>> GetObjectsAsync(Guid id, int level
                 parameters = new Dictionary<string, string>
                 {
                     { "depth", levels.ToString() },
-                    { "flatten", "true".ToString() }, //This parameter is needed to get the data in a 'flat' way and keep consistency in the logic to retrieve the objects
-                    { "includeExtensions", includeExtensions.ToString() },
-                    { "includeInternal", includeInternalObjects.ToString() } //This param has different name when version > v3
+                    { "flatten", "false" },

```

Comment:
This actually needs to be `"false"` so that we get back a tree; otherwise we have to reassemble a tree to get the property output.